helm_deploy/hmpps-restricted-patients-api/templates/past-date-check-cronjob.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: hmppps-restricted-patients-api-past-date-check
+spec:
+  schedule: "0 3 * * *"
+  suspend: true
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 5
+  startingDeadlineSeconds: 600
+  successfulJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
```

Comment:
Id suggest you include a 
`ttlSecondsAfterFinished: 345600`
or similar (this clears failed jobs after 4 days)